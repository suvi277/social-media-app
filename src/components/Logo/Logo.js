import React from 'react';

const Logo = () => {
	return (
		<a href="/" className="navbar-brand">
			<img src="https://www.kalido.me/wp-content/themes/kalido-theme/img/logo.svg" alt="Kalido" width="150" />
		</a>
	);
};
export { Logo };
