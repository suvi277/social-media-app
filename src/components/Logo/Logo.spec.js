import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from './Logo';

describe('Logo Component', () => {
	const wrapper = shallow(<Logo />);

	it('renders as expected', () => {
		expect(wrapper).toMatchSnapshot();
	});
	it('should exist h1 tag', () => {
		expect(wrapper.find('img')).toHaveLength(1);
	});
});
