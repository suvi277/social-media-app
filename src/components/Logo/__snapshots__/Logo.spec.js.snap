// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component renders as expected 1`] = `ShallowWrapper {}`;
